analyses/chromosomal-instability/util/chr-break-calculate.R

```diff
@@ -0,0 +1,233 @@
+# Functions for chromosomal instability calculations
+#
+# C. Savonen for ALSF - CCDL
+#
+# 2020
+
```

Comment:
I wonder if it makes sense to load libraries here? Otherwise, the script(s) calling these functions fails to load libraries for whatever reason the code won't break.

Reply:
All functions used used `::` so no libraries should be necessary.

Reply:
good point! great.